docs/pages/docs/docs-theme/theme-configuration.mdx

```diff
@@ -483,6 +483,25 @@ export default {
 }
 ```
 
+## Theme Switch
+
+You are able to customize the option names:
+
+```jsx
+export default {
+  themeSwitch: {
+    useOptions() {
```

Comment:
API looks weird to me, can we use 
```
type ThemeSwitchLabel = 'toggle' | 'system' | 'light' | 'dark'
themeSwitch: {
  ...
  labels: Record<ThemeSwitchLabel, string> | (...) => Record<ThemeSwitchLabel, string>
}
```

Reply:
OK I saw you are using hook style API in other places, and only for `locale`, can we simply make it as an parameter of the function like `({ locale }) => ...` or `({ router }) => ...`, don't think it's a good idea to require hook as option getter

Reply:
Nice idea! I did agree with that, it's similar to other existing solutions such as the `search` property in the docs theme

Reply:
Anyway another point in my comments is that you can simply use object instead of array for localised labels, and you'd better to provide a way to localise the toggle button's aria-label